### ISSUE TITLE
Compile error when nesting `edges` & `node` types in a GraphQL query (duplicate class names)

### ISSUE THREAD

**Issue by rsmckinney** [manifold-systems/manifold#155]
**Describe the bug**
A GraphQL query with a nested `edges` causes a compile error of the form:

>Error:java: interface abc.res.queries.RepoInfo.Result.repositoryOwner.repository.pullRequests.edges is already defined in interface abc.res.queries.RepoInfo.Result.repositoryOwner.repository.pullRequests

**To Reproduce**
Use a query with nested `edges` such as:
```graphql
query RepoInfo($login: String!, $repo: String!, $pullRequestsCount: Int, $commentsCount: Int) {
    repositoryOwner(login: $login) {
        repository(name: $repo) {
            name
            stargazers {
                totalCount
            }
            pullRequests(last: $pullRequestsCount, states: [OPEN]) {
                edges {
                    node {
                        title
                        author {
                            login
                        }
                        comments(last: $commentsCount) {
                            edges {
                                node {
                                    author {
                                        login
                                    }
                                }
                            }
                        }
                    }
                }
            }
        }
    }
}
```
**Desktop (please complete the following information):**
- Java/JDK version: 1.8
- IntelliJ IDEA version: 2020.1 eap
- Manifold version: 2019.1.32
- Manifold IntelliJ plugin version: 2020.1.0

**Comment by rsmckinney** [comment]
in progress

**Comment by rsmckinney** [comment]
Fixed in 2020.1.1 with https://github.com/manifold-systems/manifold/commit/0a5f8d2e611afc230823c85a6d90a86793841b0b
